ci/pipeline.yml

```diff
@@ -899,25 +814,20 @@ jobs:
 - name: deploy-logsearch-production
   serial_groups: [bosh-production]
   plan:
-  - aggregate:
+  - in_parallel:
     - get: pipeline-tasks
+      passed: [smoke-tests-platform-production, upload-kibana-objects-platform-production]
     - get: logsearch-config
-      passed: [deploy-logsearch-staging]
       trigger: true
     - get: common-secrets
       resource: common-prod
       trigger: true
     - get: logsearch-release
```

Comment:
why are we removing the `passed` clause here? same question for the removals around 395

Reply:
It's both a rendering change (shouldn't affect deployment timing) and a deployment ordering change. It updates the deployment ordering to look like this:

1. Deploy dev platform & smoke test
1. Deploy dev & smoke test
1. Deploy staging platform & smoke test
1. Deploy staging & smoke test
1. Deploy prod platform & smoke test
1. Deploy prod & smoke test

I want the platform cluster to deploy first so any issues are caught by our internal platform logs cluster before the customer logs cluster.

Reply:
I think you still need:
```
passed: [smoke-tests-<whatever>]
``` 
for all of the dependencies, otherwise it will look for the newest version of each dependency combined with the `pipeline-tasks` version that has passed `smoke-tests-platform-production` and `upload-kibana-objects-platform-production`

Reply:
Sure, I'll see how I can refactor it.